scripts/add_license.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
 PAT_APA="^// Copyright 2020 ChainSafe Systems // SPDX-License-Identifier: Apache-2.0$"
+PAT_CODEGEN="^// This file is generated by "
 
 valid=true
 for file in $(find . -type f -not -path "./target/*" -not -path "./blockchain/beacon/src/drand_api/*" -not -path "./ipld/graphsync/src/message/proto/message.rs" | egrep '\.(rs)$'); do
```

Comment:
I think js files need the license header as well, we can aso remove the exceptions they are from Forest

Reply:
Good suggestion, fixed